src/views/UserDetails.vue

```diff
@@ -45,7 +44,7 @@
               <products-table
                 :products="userProducts"
                 :additionalColumnsRight="disassemblyColumns"
-                :title="`${user.name}'s products table`"
+                :title="`${user.firstName}'s products table`"
```

Comment:
We could go for both first and last name here.

Reply:
fixed